packages/shared/src/hardware/instance.ts

```
@@ -25,70 +25,110 @@ export const generateConnectSrc = () => {
+
+    settings.preRelease = params.isPreRelease;
+
+    try {
+      await HardwareSDK.init(settings, HardwareLowLevelSDK);
+      // debugLogger.hardwareSDK.info('HardwareSDK initialized success');
+      console.log('HardwareSDK initialized success');
+      resolve(HardwareSDK);
+    } catch (e) {
+      reject(e);
+    }
+  });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)